packages/design-system/CONTRIBUTING.md

```diff
@@ -1,53 +1,140 @@
 # Contributing
+Congratulations on becoming a contributor to the design system! You are pushing the whole company upwards.
 
-Please use this template in order to add a new Design System element documentation.
+Here's what we expect our contributors to do when it comes to code.
 
-Each new element should have this hierarchy
+We don't believe in giving you rigid steps to follow, we prefer guidance and a loose checklist. Always put people before processes.
 
+## How do I know what I can do?
+We have a [self-service Jira board set-up for this](https://jira.talendforge.org/secure/RapidBoard.jspa?projectKey=TUX&rapidView=1030). All the tasks in the TODO column are up for grabs!
+
+Assign yourself to a ticket, it's all fair game.
+
+You can also simply ask on Slack, channel #ask-designsystem.
+
+## TL;WR
+- Contribute to the design phase by offering browser-based know-how.
+- You may need to write the initial Storybook PR for contributing designers.
+- Get peer reviews on the API of your component before moving on to the implementation.
+- Follow the basic structure of a component/layout folder in the repository and use the [provided template for doc pages](DOCTEMPLATE.md).
 ```
-├── Component.tsx
-├── Component.spec.tsx     # if you need e2e tests
-├── Component.stories.mdx
-├── Component.stories.tsx
-├── Component.style.tsx
-├── Component.test.tsx     # if you need unit tests
-├── index.ts
-└── variations
-    ├── Component.A.tsx
-    └── Component.B.tsx
+├── Component.tsx          # Component
+├── Component.spec.tsx     # Cypress tests
+├── Component.stories.mdx  # Documentation and call to `<Story>`
+├── Component.stories.tsx  # Actual stories written explicitely in TSX
+├── Component.module.scss  # Component styles (in BEM)
+├── index.ts               # Clean component export
+└── private                # Internal composition elements
+    ├── ComponentBrick.A.tsx
+    └── ComponentBrick.B.tsx
 ```
+- Use design tokens with a burning passion.
+- Explicitness > Implicitness, always and in all things.
+- Don't let consumers easily override the styles or behaviours of the component.
```

Comment:
this is not a constrain if we want to manage customer theme in the future?

Reply:
No, customer themes will come from design tokens, not classnames :)